README.md

```diff
@@ -144,28 +144,28 @@ E.g.
 
 to enable retrying for POST requests.
 
-== Testing Hooks
+## Testing Hooks
 
 The client library implements several hooks to enable/disable/trigger special
 behaviors. These hooks are triggered using environment variables:
 
 * Enable debugging mode:
 
-    $ TD_CLIENT_DEBUG=1
+    `$ TD_CLIENT_DEBUG=1`
 
   Currently debugging mode consists of:
 
-  * show request and response of +HTTP+/+HTTPS+ +GET+ REST API calls;
-  * show request of +HTTP+/+HTTPS+ +POST+/+PUT+ REST API calls.
+  * show request and response of `HTTP`/`HTTPS` `GET` REST API calls;
+  * show request of `HTTP`/`HTTPS` `POST`/`PUT` REST API calls.
 
-== 1.8 compatibility
+## 1.8 compatibility
 
 Although we are trying to keep 1.8.7 compatibility as you can see it in
 .travis.yml we don't officially support 1.8.7. The most important defect
 on 1.8 is it does not support TLS only SSL endpoint.
 
-== Copyright
-
-Copyright:: Copyright (c) 2011 Treasure Data Inc.
-License::   Apache License, Version 2.0
+## Copyright
 
+|Copyright | Copyright (c) 2011 Treasure Data Inc. |
```

Comment:
This line is not a header of a table. I think it is better to the follow (derived from https://github.com/fluent/fluentd):

Before:

# Copyright
  
|Copyright | Copyright (c) 2011 Treasure Data Inc. |
|:--|:--|
|License|   Apache License, Version 2.0 |

After:

## More Information

- Copyright: (c) 2011 Treasure Data Inc.
- License: Apache License, Version 2.0